### PR TITLE
Revise prod resources to run evaluation models

### PR DESCRIPTION
**Pull request by dchiquito**
This should resolve the issues we debugged with Jim yesterday.
@annehaley would you mind filling out the instructions for the git LFS documentation section?

### CODE REVIEW

**Review by annehaley — APPROVED** [review]
One more thing we noted from Jim's configuration is that running the zarr conversion task required even more fixing, but since he didn't need zarr (nor does any user for the time being since we don't do anything with the zarr data), we simply turned off `ZARR_SUPPORT` in `settings.py`. I'm thinking we should turn this off by default until we do something with the zarr data. Is that change appropriate for this PR?

File: prod/README.md [hunk 1]
```diff
@@ -38,7 +38,12 @@ docker run -v ~/git/miqa/prod:/etc/nginx/conf.d:ro --network=host nginx
 ```
 
 ## Pull the evaluation models from git LFS
-TODO
+```
+cd miqa/learning/models
+git-lfs fetch
```
Comment (dzenanz) [inline prod/README.md]: Do we have `git lfs install` mentioned anywhere?

**Comment by dzenanz** [comment]
ZARR_SUPPORT off by default is a reasonable addition to this PR. And it looks increasingly likely we will not add meaningful multi-scale ZARR feature before the project runs out.

File: prod/README.md (hunk 1 above)
Comment (annehaley) [reply prod/README.md]: Added in [1547442](https://github.com/OpenImaging/miqa/pull/388/commits/1547442cf85cfa61499913972a9d27b10c9281a9)

**Comment by annehaley** [comment]
> ZARR_SUPPORT off by default is a reasonable addition to this PR. And it looks increasingly likely we will not add meaningful multi-scale ZARR feature before the project runs out.

Great. I toggled the setting in [c976022](https://github.com/OpenImaging/miqa/pull/388/commits/c976022b118adc75a5a2ef1fcb478da8908c9d9e).

**Review by dzenanz — APPROVED** [review]

**Comment by dchiquito** [comment]
Thanks for adding that @annehaley

**Pull request merged**